src/text-input-autocomplete-menu.component.ts

```diff
@@ -53,6 +61,27 @@ export class TextInputAutocompleteMenuComponent {
   get choices() {
     return this._choices;
   }
+  @HostBinding('style.top.px')
```

Comment:
This will be ran on every change detection cycle and could cause perf issues, can we just calculate it once here? https://github.com/UrielCuriel/angular-text-input-autocomplete/blob/master/src/text-input-autocomplete.directive.ts#L181-L184